examples/tomo/checks/check_axes_cone3d_fp.py

```diff
@@ -0,0 +1,217 @@
+               title='Projection at 90 degrees, axis = [0, 0, 1]')
+fig, ax = plt.subplots()
+ax.imshow(sum_along_x.T, cmap='bone', origin='lower')
+ax.set_xlabel('y')
+ax.set_ylabel('z')
+plt.title('Sum along x axis')
+plt.show()
+# Check axes in geometry
+axes_sum_x = geometry.det_axes(np.deg2rad(90))
+assert np.allclose(axes_sum_x[0], [0, 1, 0])
+assert np.allclose(axes_sum_x[1], [0, 0, 1])
+
+
+# %% Test case 2: axis = [0, 1, 0] -- setup
+
+
+geometry = odl.tomo.CircularConeFlatGeometry(
+    angle_partition, detector_partition, src_radius, det_radius,
+    axis=[0, 1, 0])
+# Check initial configuration
+assert np.allclose(geometry.det_axes_init[0], [1, 0, 0])
+assert np.allclose(geometry.det_axes_init[1], [0, 1, 0])
+assert np.allclose(geometry.src_to_det_init, [0, 0, -1])
+
+# Create projections
+ray_trafo = odl.tomo.RayTransform(reco_space, geometry, impl=impl)
+proj_data = ray_trafo(phantom)
+
+
+# %% axis = [0, 1, 0], projection along z axis
+
+
+# Axes in this image are (x, y). This corresponds to:
+# axis = [0, 1, 0], 0 degrees
+proj_data.show(indices=[0, slice(None), slice(None)],
+               title='Projection at 0 degrees, axis = [0, 1, 0]')
+fig, ax = plt.subplots()
+ax.imshow(sum_along_z.T, cmap='bone', origin='lower')
+ax.set_xlabel('x')
+ax.set_ylabel('y')
+plt.title('Sum along z axis')
+plt.show()
+# Check geometry axes
+axes_sum_z = geometry.det_axes(np.deg2rad(0))
+assert np.allclose(axes_sum_z[0], [1, 0, 0])
+assert np.allclose(axes_sum_z[1], [0, 1, 0])
+
+
+# %% axis = [0, 1, 0], projection along x axis
+
+
+# Axes in this image are (z, y). This corresponds to
+# axis = [0, 1, 0], 270 degrees
+proj_data.show(indices=[3, slice(None), slice(None)],
+               title='Projection at 270 degrees, axis = [0, 1, 0]')
+fig, ax = plt.subplots()
+ax.imshow(sum_along_x, cmap='bone', origin='lower')
+ax.set_xlabel('z')
+ax.set_ylabel('y')
+plt.title('Sum along x axis, transposed')
+plt.show()
+# Check geometry axes
+axes_sum_x_T = geometry.det_axes(np.deg2rad(270))
+assert np.allclose(axes_sum_x_T[0], [0, 0, 1])
+assert np.allclose(axes_sum_x_T[1], [0, 1, 0])
+
+
+# %% Test case 3: axis = [1, 0, 0] -- setup
+
+
+geometry = odl.tomo.CircularConeFlatGeometry(
+    angle_partition, detector_partition, src_radius, det_radius,
+    axis=[1, 0, 0])
+# Check initial configuration
+assert np.allclose(geometry.det_axes_init[0], [0, 0, -1])
+assert np.allclose(geometry.det_axes_init[1], [1, 0, 0])
+assert np.allclose(geometry.src_to_det_init, [0, 1, 0])
+
+# Create projections
+ray_trafo = odl.tomo.RayTransform(reco_space, geometry, impl=impl)
+proj_data = ray_trafo(phantom)
+
+
+# %% axis = [1, 0, 0], projection along z axis
+
+
+# Axes in this image are (y, x). This corresponds to
+# axis = [1, 0, 0], 90 degrees
+proj_data.show(indices=[1, slice(None), slice(None)],
+               title='Projection at 90 degrees, axis = [1, 0, 0]')
+fig, ax = plt.subplots()
+ax.imshow(sum_along_z, cmap='bone', origin='lower')
```

Comment:
That would also require a bit of boilerplate code to create a space that fits the array. Some of that stuff is part of #861, which will simplify these examples.

For now creating a space for the sum along an axis won't really make the code simpler I guess.